config/config.php

```diff
@@ -0,0 +1,88 @@
+ * @license      GNU GPL 2 or later (https://www.gnu.org/licenses/gpl-2.0.html)
+ * @author       XOOPS Development Team
+ */
+
+$moduleDirName      = basename(dirname(__DIR__));
+$moduleDirNameUpper = mb_strtoupper($moduleDirName);
+
+return (object)[
+    'name'           => $moduleDirNameUpper . ' Module Configurator',
+    'paths'          => [
+        'dirname'    => $moduleDirName,
+        'admin'      => XOOPS_ROOT_PATH . '/modules/' . $moduleDirName . '/admin',
+        'modPath'    => XOOPS_ROOT_PATH . '/modules/' . $moduleDirName,
+        'modUrl'     => XOOPS_URL . '/modules/' . $moduleDirName,
+        'uploadPath' => XOOPS_UPLOAD_PATH . '/' . $moduleDirName,
+        'uploadUrl'  => XOOPS_UPLOAD_URL . '/' . $moduleDirName,
+    ],
+    'uploadFolders'  => [
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName,
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName . '/category',
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName . '/screenshots',
+        //XOOPS_UPLOAD_PATH . '/flags'
+    ],
+    'copyBlankFiles' => [
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName,
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName . '/category',
+        XOOPS_UPLOAD_PATH . '/' . $moduleDirName . '/screenshots',
+        //XOOPS_UPLOAD_PATH . '/flags'
+    ],
+
+    'copyTestFolders' => [
+        //        [
+        //            XOOPS_ROOT_PATH . '/modules/' . $moduleDirName . '/testdata/uploads',
+        //            XOOPS_UPLOAD_PATH . '/' . $moduleDirName,
+        //        ],
+        //            [
+        //                XOOPS_ROOT_PATH . '/modules/' . $moduleDirName . '/testdata/thumbs',
+        //                XOOPS_UPLOAD_PATH . '/' . $moduleDirName . '/thumbs',
+        //            ],
+    ],
+
+    'templateFolders' => [
+        '/templates/',
+        //            '/templates/blocks/',
+        //            '/templates/admin/'
+    ],
+    'oldFiles'        => [
+        '/class/request.php',
+        '/class/registry.php',
+        '/class/utilities.php',
+        '/class/util.php',
+        //            '/include/constants.php',
+        //            '/include/functions.php',
+        '/ajaxrating.txt',
+    ],
+    'oldFolders'      => [
+        '/images',
+        '/css',
+        '/js',
+        '/tcpdf',
+        '/images',
+    ],
+
+    'renameTables' => [//         'XX_archive'     => 'ZZZZ_archive',
+    ],
+    'moduleStats'  => [
+        //            'totalcategories' => $helper->getHandler('Category')->getCategoriesCount(-1),
+        //            'totalitems'      => $helper->getHandler('Item')->getItemsCount(),
+        //            'totalsubmitted'  => $helper->getHandler('Item')->getItemsCount(-1, [Constants::PUBLISHER_STATUS_SUBMITTED]),
+    ],
+    'modCopyright' => "<a href='https://xoops.org' title='XOOPS Project' target='_blank'>
+                     <img src='" . constant($moduleDirNameUpper . '_AUTHOR_LOGOIMG') . "' alt='XOOPS Project'></a>",
```

Comment:
Updating the module results in

Warning: constant(): Couldn't find constant TAG_AUTHOR_LOGOIMG in file /modules/tag/config/config.php line 86

Reply:
OK, I'm on it....